### DISCUSSION TITLE
Problem with copying component code

### DISCUSSION THREAD

**Discussion by Enayut** [damien-schneider/cuicui#25] [General]
![image](https://github.com/user-attachments/assets/6c429450-6c21-4a14-a3af-93b1a4c99f58)

As we can see here we have some error with reading file from path

```
An error occurred while reading file from path /var/task/src/ui/cuicui/common-ui/cards/modern-inner-shadow/variant1.tsx:
{"errno":-2,"code":"ENOENT","syscall":"open","path":"/var/task/src/ui/cuicui/common-ui/cards/modern-inner-shadow/variant1.tsx"}
```

**Comment by ionutcnu** [comment]
+1 this is happening to all components

**Comment by damien-schneider** [comment]
Thank you for reporting this ! Everything is fixed, and a test have been written to check this not happen anymore 😉
